<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-CQL946G0" failures="1" tests="3" name="loginTest" time="35.323" errors="0" timestamp="2024-06-22T10:03:30 IST">
  <testcase classname="TestNG.LoginTest" name="loginOperation" time="3.117"/>
  <testcase classname="TestNG.LoginTest" name="validateLoginByUrl" time="0.052">
    <failure type="java.lang.AssertionError" message="Validate Login By Url Failed.. expected [https://opensource-demo.orangehrmlive.com/web/index.php/dashboard/index1] but found [https://opensource-demo.orangehrmlive.com/web/index.php/dashboard/index]">
      <![CDATA[java.lang.AssertionError: Validate Login By Url Failed.. expected [https://opensource-demo.orangehrmlive.com/web/index.php/dashboard/index1] but found [https://opensource-demo.orangehrmlive.com/web/index.php/dashboard/index]
at TestNG.LoginTest.validateLoginByUrl(LoginTest.java:59)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- validateLoginByUrl -->
  <testcase classname="TestNG.LoginTest" name="validateLoginByText" time="0.0">
    <skipped/>
  </testcase> <!-- validateLoginByText -->
</testsuite> <!-- loginTest -->
